crates/bevy_render/macros/src/as_bind_group.rs

```diff
@@ -366,7 +366,7 @@ pub fn derive_as_bind_group(ast: syn::DeriveInput) -> Result<TokenStream> {
                                 if let Some(handle) = handle {
                                     let image = images.get(handle).ok_or_else(|| #render_path::render_resource::AsBindGroupError::RetryNextUpdate)?;
 
-                                    let Some(sample_type) = image.texture_format.sample_type(None, None) else {
+                                    let Some(sample_type) = image.texture_format.sample_type(None, Some(render_device.features())) else {
```

Comment:
For reference: https://docs.rs/wgpu/latest/wgpu/enum.TextureFormat.html#method.sample_type